src/QsCompiler/Tests.Compiler/SerializationTests.fs

```diff
@@ -58,7 +58,7 @@ module SerializationTests =
         Type = t |> ResolvedType.New
         InferredInformation = InferredExpressionInformation.New (false, false)
         Position = Null
-        Range = {Line = 1; Column = s}, {Line = 1; Column = e}
+        Range = Range.Create (Position.Create 1 s) (Position.Create 1 e)
```

Comment:
Since we have gone from one-based to zero-based, should the literals in this be 0 instead of 1?

Reply:
I guess I'm concerned that all the literals in this file are now one greater than they should be.

Reply:
I think you're right. It didn't matter for the test since it serializes to JSON and back based on the ranges given in the file, but updating the range positions seems reasonable.

Reply:
All of the literals for HeaderRange and SymbolRange were reduced by one, but the literals on the Position fields were not. Is that okay here?

Reply:
The position fields used to be of type `int * int`, while the ranges were `QsPositionInfo * QsPositionInfo`. I think `int * int` was zero-based and `QsPositionInfo` was one-based. (It's really confusing...) So I only subtracted 1 from the ones that used to be `QsPositionInfo`s.